extensions/adobe/experience/marketo/marketo-contact-lead-person.schema.json

```diff
@@ -0,0 +1,190 @@
+          "description": "Metropolitan area inferred by reverse IP lookup of lead’s first recorded web visit.",
+          "type": "string"
+        },
+        "marketo:inferredPhoneAreaCode": {
+          "title": "Inferred Phone Area Code",
+          "description": "Phone area code inferred by reverse IP lookup of lead’s first recorded web visit.",
+          "type": "string"
+        },
+        "marketo:anonymous": {
+          "title": "Anonymous Indicator",
+          "description": "Anonymous status of lead record. System managed.",
+          "type": "boolean"
+        },
+        "marketo:customer": {
+          "title": "Customer Indicator",
+          "description": "Customer indicator.",
+          "type": "boolean"
+        },
+        "marketo:employee": {
+          "title": "Employee Indicator",
+          "description": "Employee indicator.",
+          "type": "boolean"
+        },
+        "marketo:partner": {
+          "title": "Partner Indicator",
+          "description": "Partner indicator.",
+          "type": "boolean"
+        },
+        "marketo:private": {
+          "title": "Private Indicator",
+          "description": "Private indicator.",
+          "type": "boolean"
+        },
+        "marketo:accountID": {
+          "title": "Marketo Account id",
+          "description": "Marketo account id.",
+          "type": "string"
+        },
+        "marketo:marketingSuspended": {
+          "title": "Marketing Suspended",
+          "description": "Marketing suspended.",
+          "type": "boolean"
+        },
+        "marketo:marketingSuspendedCause": {
+          "title": "Marketing Suspended Cause",
+          "description": "Marketing suspended cause.",
+          "type": "string"
+        },
+        "marketo:notes": {
+          "title": "Notes",
+          "description": "Notes.",
+          "type": "string"
+        },
+        "marketo:orgID": {
+          "title": "Organization ID",
+          "description": "Organization id.",
+          "type": "string"
+        },
+        "marketo:primaryLeadInterestID": {
+          "title": "Primary Lead Interest Id",
+          "description": "Primary lead interest id.",
+          "type": "boolean"
+        },
+        "marketo:registerSourceInfo": {
+          "title": "Register Source Info",
+          "description": "Register source info.",
+          "type": "string"
+        },
+        "marketo:registerSourceType": {
+          "title": "Register Source Type",
+          "description": "Register source type.",
+          "type": "string"
+        },
+        "marketo:salesContactOwnerID": {
+          "title": "Sales Contact Owner Id",
+          "description": "Sales contact owner id.",
+          "type": "string"
+        },
+        "marketo:originalReferrer": {
+          "title": "Original Referrer",
+          "description": "The original referrer is typically the page that leads to a form fill out.",
+          "type": "string"

```

Comment:
so is this a URL?  If so, let's validate it as such?

Reply:
It is defined as "string" in MARKETO